scripts/Player.gd

```diff
@@ -8,9 +8,22 @@ const JUMP_HEIGHT_LOW = -420
 const JUMP_HEIGHT_HIGH = -550
 
 var motion = Vector2()
+var apply_friction = false
```

Comment:
Could you please do `_apply_friction`? Also `_motion`? These are leftovers from my old project import before we had any guidelines.